platform/cluster.go

```diff
@@ -162,6 +162,17 @@ func (bc *BaseCluster) RenderUserData(userdata *conf.UserData, ignitionVars map[
 		conf.CopyKeys(keys)
 	}
 
+	if bc.OSContainer() != "" {
+		if bc.Distribution() != "rhcos" {
+			return nil, fmt.Errorf("oscontainer is only supported on the rhcos distribution")
+		}
+		conf.AddSystemdUnitDropin("pivot.service", "00-before-sshd.conf", `[Unit]
+Before=sshd.service`)
+		conf.AddSystemdUnit("pivot.service", "", true)
```

Comment:
Missing body?

Reply:
This is intentional, I'm just enabling the `pivot.service` unit